packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.java

```diff
@@ -78,7 +78,13 @@ public void onCreate(ModuleRegistry moduleRegistry) {
   @ExpoMethod
   public void reload(final Promise promise) {
     try {
-      UpdatesController.getInstance().relaunchReactApplication(getContext(), new Launcher.LauncherCallback() {
+      UpdatesController controller = UpdatesController.getInstance();
+      if (!controller.getUpdatesConfiguration().isEnabled()) {
+        promise.reject("ERR_UPDATES_RELOAD", "You cannot reload when expo-updates is not enabled.");
```

Comment:
The cause of the error is that updates are disabled, so I think we want to reflect that here (rather than listing the name of the operation that failed):

```suggestion
        promise.reject("ERR_UPDATES_DISABLED", "You cannot reload when expo-updates is not enabled.");
```